Apps/Admin/Server/Controllers/AdminReportController.cs

```diff
@@ -67,6 +68,9 @@ public async Task<IEnumerable<BlockedAccessRecord>> BlockedAccessReport(Cancella
         /// <summary>
         /// Retrieves a collection of user HDIDs that have dependents.
         /// </summary>
+        /// <param name="page">Page number of the protected dependents report.</param>
```

Comment:
I think it should be documented whether page is indexed starting at 0 or 1.

Reply:
Thanks I have to change it to zero indexing to be in line with MudTable's state property page.